fml/raster_thread_merger.cc

```diff
@@ -48,11 +62,17 @@ void RasterThreadMerger::ExtendLeaseTo(size_t lease_term) {
 }
 
 bool RasterThreadMerger::IsMerged() const {
-  return is_merged_;
+  return is_merged_.load();
+}
+
+void RasterThreadMerger::WaitUntilMerged() {
+  FML_CHECK(MessageLoop::GetCurrentTaskQueueId() != gpu_queue_id_);
+  while (!is_merged_.load())
```

Comment:
Instead of using a spin-lock, use a condition variable and a boolean predicate check.

Reply:
done